ztag/transforms/ftp.py

```diff
@@ -17,12 +18,12 @@ def _transform_object(self, obj):
         ftp = Transformable(obj)
         zout = ZMapTransformOutput()
         error = ftp['error'].resolve()
-        if error is not None:
+        if error != None:
```

Comment:
Why did these get changed? Also, per PEP8, comparisons to singletons (such as `None`), should always be done with the `is` operator. https://www.python.org/dev/peps/pep-0008/#programming-recommendations